modules/codebuild/codebuild.tf

```diff
@@ -30,7 +30,7 @@ resource "aws_codebuild_project" "codebuild" {
     type = "GITHUB"
 
     location        = "https://github.com/${local.git_config[0].organization_name}/${local.git_config[0].repo_name}.git"
-    buildspec       = local.get_codebuildspec_file
+    buildspec       = local.get_codebuildspec_logic
```

Comment:
what is a reason to change variable name?

Reply:
It's changing the logic of the whole module a bit, same like you did for multirepo module. It allows you essentially to override the build spec file